deploy_openshift_aio.sh

```diff
@@ -47,8 +48,20 @@ rm -r $OC_DIR $OC_TARFILE
 cd $SCRIPT_DIR
 oc version
 
-title "Deploying OpenShift AIO cluster"
+title "Initializing OpenShift AIO cluster"
 oc cluster up --service-catalog=true --public-hostname=$(hostname -f)
```

Comment:
You can probably expand on the logic here so that you don't have to `oc cluster up` twice, but, for now, this does the job of bootstrapping the configs, and then altering them.